test/clojure/core_test/eq.cljc

```diff
@@ -0,0 +1,116 @@
+(ns clojure.core-test.eq
+  (:require [clojure.test :as t]))
+
+;; Doesn't test (yet):
+;; * Java collections for JVM Clojure
+;; * clojure.lang.PersistentQueue
+;; * ##Inf and some other special forms
+;; * Records
+
+(t/deftest two-scalars
+  (t/are [in ex] (= in ex)
+    nil nil
+    true true
+    false false
+    \a \a
+    "green" "green"
+    :eggs :eggs
+    :and/ham :and/ham
+    'one-fish 'one-fish
+    'two/fish 'two/fish
+    42 42
+    84/2 42
```

Comment:
`84/2` actually reads as `42`, not a ratio. If we're trying to compare a ratio to a number, we'll need something which reads as a ratio.

Reply:
Good catch; I switched to 22/7 instead